zerver/models.py

```diff
@@ -610,6 +610,7 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
     muted_topics = models.TextField(default=u'[]') # type: Text
 
     objects = UserManager() # type: UserManager
+    quota = models.IntegerField(default=1073741824) # type: int
```

Comment:
You could also split this out to a constant

`UserProfile.DEFAULT_UPLOADS_QUOTA` that can then be accessed directly by the test below...